linera-execution/src/system.rs

```diff
@@ -172,6 +176,8 @@ pub enum SystemOperation {
     ProcessNewEpoch(Epoch),
     /// Processes an event about a removed epoch and committee.
     ProcessRemovedEpoch(Epoch),
+    /// Updates the event stream trackers.
+    UpdateStreams(Vec<(ChainId, StreamId, u32)>),
```

Comment:
I can't find where this is emitted 👀

Reply:
It isn't. That will follow in the next PR.

Reply:
OK, slightly confusing it's here now.

Reply:
Yes; I was trying to minimize the diff of the upcoming PR, because that will already contain somewhat complicated code. But I could remove it for now, if you prefer.

Reply:
It's easier to review stuff when there's a context for its usage.